### PR TITLE
Issue #108 Change Title of Save As File Chooser

### PR DESCRIPTION
**Pull request by klhes**
Changed title of File Chooser from 'Save' to 'Save As'.

### CODE REVIEW

File: OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java [hunk 1]
```diff
@@ -67,9 +67,11 @@ public static File getSelectedFile(String extension, String description, DialogT
         int returnVal = 0;
         switch (type) {
             case SAVE:
+                chooser.setDialogTitle("Save As");
```
Comment (sebbrudzinski) [inline OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java]: Why do we set title to "Save as" both for SAVE and SAVEAS case?

Comment (klhes) [inline OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java]: That's the convention that I'm used to. Clicking 'Save' when a new/unsaved file is open in Word, Excel, etc. will open a dialog with the title 'Save As'. 

If you agree with using the convention but think the code is unclear, I can try to make it more readable.  For example, in the 'getFileWithConfirmation' method, I could change the type to SAVEAS when file == null.

Comment (sebbrudzinski) [inline OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java]: Actually, I think I'm fine with leaving it as it is. It makes me wonder though, why we have separated SAVE and SAVEAS cases here. I don't think the behaviour between those two should ever be different. Can we merge those cases into one?

Comment (klhes) [inline OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java]: The behavior of the SAVE and SAVEAS cases is different in the getFileWithConfirmation method.  If the type is SAVEAS, the getSelectedFile method is always called.  If the type is SAVE, the getSelectedFile method is only called when currentFile is null.  


```
      public static File getFileWithConfirmation(File currentFile, String extension, String description, DialogType type, boolean fixExtension) {
        File file = currentFile;
        if (file == null || type == DialogType.SAVEAS) {
            file = getSelectedFile(extension, description, type, fixExtension);
        }
        
        file = confirmFileSave(file, currentFile);
        
        return file;
}
```

Comment (sebbrudzinski) [inline OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java]: What I meant is why we have separated it in this method, inside the switch loop. We can merge the cases like
```
case SAVE:
case SAVEAS:
    <code>
```

Comment (klhes) [inline OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java]: Sorry I misunderstood!  You're correct; there's no reason to keep those cases separate in the switch statement.  I'll merge them.

**Pull request merged**